bar/default.nix

```diff
@@ -1,20 +1,19 @@
 { self, ... }: {
-  perSystem = { self', inputs', system, pkgs, ... }:
+  perSystem = { self', config, inputs', system, pkgs, ... }:
     let
-      purs-nix = self.inputs.purs-nix { inherit system; };
-      dependencies = with purs-nix.ps-pkgs; [
+      dependencies = with config.purs-nix.ps-pkgs; [
         prelude
         effect
         console
-        self'.packages.foo
+        foo
```

Comment:
This is one clear benefit of the new overlay system. Because, I can refer to both local packages and overridden packages _uniformly_.